Berksfile

```diff
@@ -0,0 +1,7 @@
+source 'https://supermarket.getchef.com'
```

Comment:
What was the reason for adding berkshelf?  This cookbook doesn't appear to have any dependencies in the metadata.  All the berkshelf changes just seem completely separate from the actual thing we want to fix here.